lib/ansible/modules/cloud/podman/podman_container.py

```diff
@@ -0,0 +1,1196 @@
+            cmd += ['--subuidname', params['subuidname']]
+        if params.get('sysctl'):
+            cmd += ['--sysctl', params['sysctl']]
+        if params.get('systemd') is not None:
+            cmd += ['--systemd=' + str(params['systemd'])]
+        if params.get('tmpfs'):
+            cmd += ['--tmpfs', params['tmpfs']]
+        if params.get('tty') is not None:
+            cmd += ['--tty=' + str(params['tty'])]
+        if params.get('uidmap'):
+            cmd += ['--uidmap', params['uidmap']]
+        if params.get('ulimit'):
+            cmd += ['--ulimit', params['ulimit']]
+        if params.get('user'):
+            cmd += ['--user', params['user']]
+        if params.get('userns'):
+            cmd += ['--userns', params['userns']]
+        if params.get('uts'):
+            cmd += ['--uts', params['uts']]
+        if params.get('volume'):
+            for vol in params['volume']:
+                cmd += ['--volume', vol]
+        if params.get('volumes_from'):
+            for vol in params['volumes_from']:
+                cmd += ['--volumes-from', vol]
+        if params.get('workdir'):
+            cmd += ['--workdir', params['workdir']]
+        # Add your own args for podman command
+        if params.get('cmd_args'):
+            cmd += params['cmd_args']
+        cmd.append(params['image'])
+        if params.get('command'):
+            cmd += params['command'].split()
+        return cmd
+
+
+def ensure_image_exists(module, image):
+    rc, out, err = run_podman_command(module, args=['image', 'exists', image], ignore_errors=True)
+    if rc == 0:
+        return
+    rc, out, err = run_podman_command(module, args=['image', 'pull', image], ignore_errors=True)
+    if rc != 0:
+        module.fail_json(msg="Can't pull image %s" % image, stdout=out, stderr=err)
+    return True
+
+
+class PodmanContainer:
+    def __init__(self, module, name, params):
+
+        super(PodmanContainer, self).__init__()
+        self.module = module
+        self.name = name
+        self.info = self.get_info()
+
+    @property
+    def exists(self):
+        return bool(self.info != {})
+
+    @property
+    def different(self):
+        # TODO(sshnaidm): implement difference calculation between input vars
+        # and current container to understand if we need to recreate it
+        return True
+
+    @property
+    def running(self):
+        return self.exists and self.info['State']['Running']
+
+    @property
+    def stopped(self):
+        return self.exists and not self.info['State']['Running']
+
+    def get_info(self):
+        rc, out, err = run_podman_command(
+            module=self.module,
+            args=['container', 'inspect', self.name], ignore_errors=True)
+        return json.loads(out)[0] if rc == 0 else {}
+
+    def _perform_action(self, action):
+        command = construct_command_from_params(action, self.module.params)
+        self.module.log("PODMAN-DEBUG: " + " ".join(command))
```

Comment:
plz use string interpolation
```suggestion
        self.module.log("PODMAN-DEBUG: %s" % " ".join(command))
```

Reply:
fixed